scripts/save-stdcm-request.py

```diff
@@ -0,0 +1,191 @@
+#! /usr/bin/env -S uv run --script
```

Comment:
We don't have an uv files so this shebang is weird.
I suggest to change it for this:


```suggestion
#!/usr/bin/env python3
```

Reply:
That's the point, `uv` can run standalone scripts without external files. Dependencies are described in the comment below and are automatically installed in a new env upon running the script.

It's still *possible* to run the script as a standard python file (if someone doesn't want to install `uv`), but that requires manually setting up and environment.

Reply:
Okay, I did not know this feature of uv.